lib/Auth/Process/privacyidea.php

```diff
@@ -131,7 +107,11 @@ public function process(&$state)
     public static function authenticate(array &$state, $otp)
     {
 
-    	$cfg = $state['privacyidea:privacyidea'];
+    	if (isset($state['privacyidea:serverconfig'])) {
```

Comment:
I think we can now access _only_ the ``privacyidea:privacyidea`` config, since we fetch the values from ``privacyidea:serverconfig`` in the method `process`.

Reply:
You're right. See https://github.com/privacyidea/simplesamlphp-module-privacyidea/commit/9fb52415eccf140ca1b718d5a66c17c778669686